lib/evaluate/call.cc

```diff
@@ -42,15 +42,15 @@ ActualArgument &ActualArgument::operator=(Expr<SomeType> &&expr) {
 }
 
 std::optional<DynamicType> ActualArgument::GetType() const {
-  if (const auto *expr{GetExpr()}) {
+  if (const auto *expr{UnwrapExpr()}) {
```

Comment:
I'd find this easier to understand if "auto" were replaced by the actual type.

Reply:
You bet.